### PR TITLE
[type:feat]: auto download MySQL connector and update configurations

### PR DESCRIPTION
**Pull request by aFlyBird0**
## Feature
* update shenyu docker image version to `2.5.0`
* use `initContainer` to download `MySQL` connector and create volume
* application.yaml of admin and bootstrap update to date

## Screenshots

image

image

image

image

image

### CODE REVIEW

**Comment by aFlyBird0** [comment]
@erdengk Please Review, thank you! ❤️

**Comment by erdengk** [comment]
@aFlyBird0 

I think we should delete the todo list here.


[delete todo](https://github.com/apache/shenyu-helm-chart/blob/0a83b638290d0222d029171ccdb4ec5cd16ef44f/charts/shenyu/README.md#shenyu-helm-chart)

image

**Comment by aFlyBird0** [comment]
> @aFlyBird0
> 
> I think we should delete the todo list here.
> 
> [delete todo](https://github.com/apache/shenyu-helm-chart/blob/0a83b638290d0222d029171ccdb4ec5cd16ef44f/charts/shenyu/README.md#shenyu-helm-chart)
> 
> image

**Comment by aFlyBird0** [comment]
> > @aFlyBird0
> > I think we should delete the todo list here.
> > [delete todo](https://github.com/apache/shenyu-helm-chart/blob/0a83b638290d0222d029171ccdb4ec5cd16ef44f/charts/shenyu/README.md#shenyu-helm-chart)
> > image

Done, thank you.

File: charts/shenyu/values.yaml [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+#################################################
+#                global configs                 #
+#################################################
+# it's not recommended to change this version
+version: 2.5.0
 replicas: 1
 admin:
   enabled: true
   image: apache/shenyu-admin
+  # if you want to change `server.port`, please edit here
```
Comment (erdengk) [inline charts/shenyu/values.yaml]: maybe 'admin service port' is better

File: charts/shenyu/values.yaml [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
   nodePort: 31095
 bootstrap:
   enabled: true
   image: apache/shenyu-bootstrap
+  # if you want to change `server.port`, please edit here
```
Comment (erdengk) [inline charts/shenyu/values.yaml]: maybe 'bootstrap service port' is better

File: charts/shenyu/values.yaml [hunk 3]
```diff
@@ -21,9 +32,496 @@ dataSource:
...
     driverClass: com.mysql.cj.Driver
+    connectorVersion: 8.0.23
+
+#################################################
+#     application.yml of admin and bootstrap    #
+#################################################
+
+applicationConfig:
+  bootstrap:
+    # Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (erdengk) [inline charts/shenyu/values.yaml]: The license does not need to be put here, it has nothing to do with values.yml.

File: charts/shenyu/values.yaml [hunk 4]
```diff
@@ -21,9 +32,496 @@ dataSource:
...
+    logging:
+      level:
+        root: info
+        org.springframework.boot: info
+        org.apache.ibatis: info
+        org.apache.shenyu.bonuspoint: info
+        org.apache.shenyu.lottery: info
+        org.apache.shenyu: info
+  admin:
+    # Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (erdengk) [inline charts/shenyu/values.yaml]: remove this

File: charts/shenyu/values.yaml (hunk 3 above)
Comment (aFlyBird0) [reply charts/shenyu/values.yaml]: Done, thanks.

File: charts/shenyu/values.yaml (hunk 1 above)
Comment (aFlyBird0) [reply charts/shenyu/values.yaml]: I agree with you 👏

**Comment by erdengk** [comment]
I tested the latest helm chart on a virtual machine and it works fine.
I think this pr can be merged, what do you think?
@JooKS-me 

image

**Review by JooKS-me — APPROVED** [review]

**Pull request merged**